mp2-v1/tests/common/cases/indexing.rs

```diff
@@ -607,7 +724,24 @@ impl TableIndexing {
             if table_row_updates.is_empty() {
                 continue;
             }
+            // If we are dealing with receipts we need to remove everything already in the row tree
             let bn = ctx.block_number().await as BlockPrimaryIndex;
+
+            let table_row_updates = if let ChangeType::Receipt(..) = ut {
+                let current_row_epoch = self.table.row.current_epoch();
+                let current_row_keys = self
+                    .table
+                    .row
+                    .keys_at(current_row_epoch)
+                    .await
+                    .into_iter()
+                    .map(TableRowUpdate::<BlockPrimaryIndex>::Deletion)
+                    .collect::<Vec<_>>();
+                [current_row_keys, table_row_updates].concat()
```

Comment:
I'm not a fan of this tbh - could we rather make the test create a new row tree ? In general DQ would follow what is implemented on the integrated test and it doesn't seem like a good way forward when we go at scale (400k - 1M entries). 
Just changing the row to be a fresh one is anyway something we wanna do for offchain data - im havent' checked that part so unclear how it's done now but that doesn't remove the goal.